relabel/relabel.go

```diff
@@ -88,6 +88,22 @@ func relabel(labels model.LabelSet, cfg *config.RelabelConfig) model.LabelSet {
 			}
 		}
 		labels = out
+	case config.RelabelLabelDrop:
+		out := make(model.LabelSet, 0)
```

Comment:
Make the cap here `len(labels)`, it'll be slightly faster as it'll avoid having to expand the slice.